headers.log

```diff
@@ -1,51 +0,0 @@
-- [x] sgx_attributes.h
```

Comment:
this file shouldn't have been added last updated it was my notes to ensure the headers were synced and it snuck in.